src/pocketdb/repositories/web/WebRpcRepository.cpp

```diff
@@ -2308,7 +2308,6 @@ namespace PocketDb
             {
                 UniValue record(UniValue::VOBJ);
 
-                auto[ok0, id] = TryGetColumnInt64(*stmt, 0);
```

Comment:
I assume that it would be better to fill the answer with this field because the client might need it. Or remove selecting this column from the query as well.
Btw, i suggest not doing such minor changes in sql because it would potentially cause a merge conflict with #502

Reply:
I see, rolled it back for now